include/rocksdb/env.h

```diff
@@ -136,6 +136,16 @@ class Env {
                                  unique_ptr<WritableFile>* result,
                                  const EnvOptions& options) = 0;
 
+  virtual Status ReuseWritableFile(const std::string& fname,
+				   const std::string& old_fname,
+				   unique_ptr<WritableFile>* result,
+				   const EnvOptions& options) {
+    Status s = RenameFile(old_fname, fname);
+    if (!s.ok())
+      return s;
```

Comment:
We follow Google C++ style in this code space, so it would be
  if (!s.ok()) {
    return s;
  }